src/StudentProfileEndorsementSchema.js

```diff
@@ -0,0 +1,96 @@
+//
+// # StudentProfileEndorsementSchema
+//
+// This schema describes the endorsements Laboratoria staff (and hopefully 3rd
+// parties) give our students. These are shown under the "Recommendations"
+// section in the Job Placement app: https://app.talento.laboratoria.la/
+//
+// This data is managed by cohort admins via the cohort users management page in
+// the LMS.
+//
+// Indexes:
+//
+// * `student`: Basic index to enhance filtering by `student`. Note that this
+//   index is NOT _unique_, which means that a given student can have many
+//   endorsements.
+// * `endorsedBy`: Basic index to enhance filtering by `endorsedBy`. Note that
+//   this index is NOT _unique_, which means that a given user can endorse many
+//   students.
+// * `organization`: Basic index to enhance filtering by `organization`. Note
+//   that this index is NOT _unique_.
+// * `student/endorsedBy`: A _unique_ compound index that ensures that a
+//   given user can only endorse a given student once.
+//
+// Migration notes:
+//
+// * `student`: Previously the _key_ in the main `user` object.
+// * `endorsedBy`: Previously we had the keys (uids) in the `recommendations`
+//   object as well as two text fields (`author` and `authorLinkedin`) and now
+//   we only store an ObjectId pointing to the author's
+//   `User` document in the database, which in turn contains the user's name and
+//   linkedin as well as other user details.
+// * `organization`: Previously we had two text fields (`company` and
+//   `companyUrl`) and now we store a ObjectId pointing to the organization
+//   document in the new `organizations` collection, which in turn contains the
+//   organization's name and url.
+// * `text`: Previously known as `detail`.
+//
+// Example old recommendations in Firestore:
+//
+// "recommendations": {
+//   "G9MyfnFrC8ViXCl3MQjX6bnqeEO2": {
+//     "author": "Lupo"
+//     "authorLinkedin": "lupomontero",
+//     "company": "Laboratoria",
+//     "companyUrl": "www.laboratoria.la",
+//     "detail": "Una recomendación...",
+//   }
+// }
+//
+// Example new student profile endorsements:
+//
+// [
+//   {
+//     "student": "ObjectId",
+//     "endorsedBy": "ObjectId",
+//     "organization": "ObjectId",
+//     "text": "Blah blah blah..."
+//   }
+// ]
+//
+
+module.exports = (conn) => {
+  const StudentProfileEndorsementSchema = new conn.Schema({
+    student: {
```

Comment:
If we change the schema name to `JobPlacementProfileEndorsementSchema`, would it make sense to rename this _field_ to `user` or even `endorsee`... :thinking: 

cc/ @gmoura

Reply:
@lupomontero At this lifecycle stage, I am not sure if sticking with the term `student` makes sense. What do you think of using the term 'graduate'? With this change, we don't have to use `JobPlacementProfileEndorsementSchema` but just `GraduateProfileEndorsementSchema`.

I might be missing something, feel free to correct me if I am wrong.

/cc @gmoura

Reply:
`StudentProfile` has now been renamed to `GraduateProfile` :wink: